.travis.yml

```diff
@@ -32,6 +32,8 @@ addons:
   apt:
     packages:
     - python2.7
+    - python-setuptools
+    - python-dill
```

Comment:
This change should not require of adding additional packages here. setuptools is already part of the Python distribution for  Python 2 >=2.7.9 and setup.py requires (will install) dill.